newIDE/app/src/MainFrame/Providers.js

```diff
@@ -113,19 +114,21 @@ const Providers = ({
                                               >
                                                 <CreditsPackageStoreStateProvider>
                                                   <ProductLicenseStoreStateProvider>
-                                                    <ExtensionStoreStateProvider>
-                                                      <BehaviorStoreStateProvider>
-                                                        <TutorialStateProvider>
-                                                          <AnnouncementsFeedStateProvider>
-                                                            <PrivateAssetsAuthorizationProvider>
-                                                              {children({
-                                                                i18n,
-                                                              })}
-                                                            </PrivateAssetsAuthorizationProvider>
-                                                          </AnnouncementsFeedStateProvider>
-                                                        </TutorialStateProvider>
-                                                      </BehaviorStoreStateProvider>
-                                                    </ExtensionStoreStateProvider>
+                                                    <MarketingPlansStoreStateProvider>
```

Comment:
moved marketing plans to its own context, allowing to display the dialog or the packages on the game page instantly, and just have the buttons load while we check if the game currently has any packages running

Reply:
Sounds good